CONTRIBUTING.md

```diff
@@ -0,0 +1,9 @@
+# Contributing
+
+We encourage any form of contribution, whether that be issues, comments, or pull requests. If you are going to be submitting a PR, there are a few things we would appreciate that you do to keep the codebase clean:
```

Comment:
For the last sentence, what do you think of `If you plan to submit a Pull Request, please follow these guidelines to help keep the codebase clean`?